extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -95,7 +96,12 @@ public NamespaceLookupExtractorFactory(
   public boolean start()
```

Comment:
I'm less comfortable with assuming the thread interrupted state is properly handled; not even JUnit clears thread interrupted state after test methods complete, so if you set the current thread's interrupted flag in a test and forget to clear it, weird things can happen.